sdk/quota/arm-quota/samples-group-quota/AddSubscriptionGroupQuota.js

```diff
@@ -19,10 +19,9 @@ require("dotenv").config();
  * x-ms-original-file: specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/examples/GroupQuotasSubscriptions/PutGroupQuotasSubscription.json
  */
 async function groupQuotaSubscriptionsPutSubscriptions() {
```

Comment:
We would not recommand to change sample folder name and this would introduce failures with some commands based on it.

https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/steps-after-generations.md#how-to-write-samples

Could you run command like below and this would help build JS and TS samples?

```
npm install -g common/tools/dev-tool # make sure you are in the azure-sdk-for-js repo root directory
cd ${PROJECT_ROOT}
npx dev-tool samples publish -f
```